
* LIB_INSTALL_DIR must be set as a relative path.
  If not, side-effect on shiboken2/CMakeList.txt: LIB_INSTALL_DIR is redefined,
  so generated Shiboken2Targets.cmake contains absolute paths that are wrong for crossbuild.

* bin/shiboken2 localization must be absolute for crossbuild.

--- sources/shiboken2/ApiExtractor/CMakeLists.txt.ORIG
+++ sources/shiboken2/ApiExtractor/CMakeLists.txt
@@ -84,7 +84,7 @@
 
 target_compile_definitions(apiextractor PRIVATE CMAKE_CXX_COMPILER="${CMAKE_CXX_COMPILER}")
 
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}" CACHE PATH "The subdirectory relative to the install prefix where libraries will be installed (default is /lib${LIB_SUFFIX})" FORCE)
+set(LIB_INSTALL_DIR "lib${LIB_SUFFIX}" CACHE PATH "The subdirectory relative to the install prefix where libraries will be installed (default is /lib${LIB_SUFFIX})" FORCE)
 
 if (BUILD_TESTS)
     find_package(Qt5Test 5.12 REQUIRED)
--- sources/shiboken2/generator/CMakeLists.txt.ORIG
+++ sources/shiboken2/generator/CMakeLists.txt
@@ -30,7 +30,7 @@
 
 install(TARGETS shiboken2
         EXPORT Shiboken2Targets
-        DESTINATION bin)
+        DESTINATION "${CMAKE_INSTALL_PREFIX}/bin")
 
 set(shiboken_generator_package_name "shiboken2_generator")
 
